examples/gcp-lakehouse/modules/gcp - sa provisionning/init.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_providers {
+
+    google = {
+      source  = "hashicorp/google"
+      version = "4.47.0"
```

Comment:
it's not recommended to use version specification inside the module, or at lease provide ranges: https://developer.hashicorp.com/terraform/language/modules/develop/providers#provider-version-constraints-in-modules

Reply:
OK - removed version altogether as I don't think it would be highly impactful